### PR TITLE
Create Python Changelog 3.8.md

### PR DESCRIPTION
**Pull request by Xiphias5985**
Adding beginning of python 3.8 changelog with proper formatting.

### CODE REVIEW

File: docs/Splashkit/Extensions/Python Compatibality/Research & Findings/Python Changelog 3.8.md [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+if (n := len(a)) \&gt; 10: print(f&quot;List is too long ({n} elements, expected \&lt;= 10)&quot;)
+
+**Positional-only parameters** introduces new notation requirements, using /, to indicate that some
+function parameters require positional specificity and cannot be used as arguments. The following
+code uses the example that parameters _a_ and _b_ are positional-only, with _c_ and _d_ being
+positional or keyword, and _e_ and _f_ being keyword-only.
+
+def (a, b, /, c, d, \*, e, f):
+
+print(a, b, c, d, e, f)
```
Comment (tancnle) [inline docs/Splashkit/Extensions/Python Compatibality/Research & Findings/Python Changelog 3.8.md:29]: **suggestion:** Should this be wrapped in a code [block](https://docs.github.com/en/get-started/writing-on-github/working-with-advanced-formatting/creating-and-highlighting-code-blocks#syntax-highlighting)? 🤔

File: docs/Splashkit/Extensions/Python Compatibality/Research & Findings/Python Changelog 3.8.md [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+Python project team, we were informed that versions prior to 3.8 functioned as intended with the
+translator. This leads to the thinking that the Python 3.8 update is specifically responsible for
+the failure in compatibility.
+
+## Python 3.8
+
+**Assignment expression** introduces new syntax, used by :=, to assign values to variables as part
+of a larger expression.
+
+if (n := len(a)) \&gt; 10: print(f&quot;List is too long ({n} elements, expected \&lt;= 10)&quot;)
```
Comment (tancnle) [inline docs/Splashkit/Extensions/Python Compatibality/Research & Findings/Python Changelog 3.8.md:20]: **suggestion:** Should this be wrapped in a code [block](https://docs.github.com/en/get-started/writing-on-github/working-with-advanced-formatting/creating-and-highlighting-code-blocks#syntax-highlighting)? 🤔

File: docs/Splashkit/Extensions/Python Compatibality/Research & Findings/Python Changelog 3.8.md (hunk 1 above)
Comment (Xiphias5985) [reply docs/Splashkit/Extensions/Python Compatibality/Research & Findings/Python Changelog 3.8.md]: Yes, they should be. I have updated both accordingly. Appreciate the feedback.

File: docs/Splashkit/Extensions/Python Compatibality/Research & Findings/Python Changelog 3.8.md [hunk 3]
```diff
@@ -0,0 +1,41 @@
...
+if (n := len(a)) > 10: 
+print(f"List is too long ({n} elements, expected <= 10)")
+```
+
+**Positional-only parameters** introduces new notation requirements, using /, to indicate that some
+function parameters require positional specificity and cannot be used as arguments. The following
+code uses the example that parameters _a_ and _b_ are positional-only, with _c_ and _d_ being
+positional or keyword, and _e_ and _f_ being keyword-only.
+
+```Python
```
Comment (tancnle) [inline docs/Splashkit/Extensions/Python Compatibality/Research & Findings/Python Changelog 3.8.md:30]:
```suggestion
```python
```

File: docs/Splashkit/Extensions/Python Compatibality/Research & Findings/Python Changelog 3.8.md [hunk 4]
```diff
@@ -0,0 +1,41 @@
...
+Python project team, we were informed that versions prior to 3.8 functioned as intended with the
+translator. This leads to the thinking that the Python 3.8 update is specifically responsible for
+the failure in compatibility.
+
+## Python 3.8
+
+**Assignment expression** introduces new syntax, used by :=, to assign values to variables as part
+of a larger expression.
+
+```Python
```
Comment (tancnle) [inline docs/Splashkit/Extensions/Python Compatibality/Research & Findings/Python Changelog 3.8.md:20]:
```suggestion
```python
```

File: docs/Splashkit/Extensions/Python Compatibality/Research & Findings/Python Changelog 3.8.md (hunk 3 above)
Comment (Xiphias5985) [reply docs/Splashkit/Extensions/Python Compatibality/Research & Findings/Python Changelog 3.8.md]: Thank you again :)

File: docs/Splashkit/Extensions/Python Compatibality/Research & Findings/Python Changelog 3.8.md [hunk 5]
```diff
@@ -0,0 +1,41 @@
...
+```
+
+**Positional-only parameters** introduces new notation requirements, using /, to indicate that some
+function parameters require positional specificity and cannot be used as arguments. The following
+code uses the example that parameters _a_ and _b_ are positional-only, with _c_ and _d_ being
+positional or keyword, and _e_ and _f_ being keyword-only.
+
+```Python
+def (a, b, /, c, d, \*, e, f):
+print(a, b, c, d, e, f)
```
Comment (tancnle) [inline docs/Splashkit/Extensions/Python Compatibality/Research & Findings/Python Changelog 3.8.md:32]: **suggestion:** Can we indent this line? I don't think this is a valid syntax.

File: docs/Splashkit/Extensions/Python Compatibality/Research & Findings/Python Changelog 3.8.md [hunk 6]
```diff
@@ -0,0 +1,41 @@
...
+print(f"List is too long ({n} elements, expected <= 10)")
+```
+
+**Positional-only parameters** introduces new notation requirements, using /, to indicate that some
+function parameters require positional specificity and cannot be used as arguments. The following
+code uses the example that parameters _a_ and _b_ are positional-only, with _c_ and _d_ being
+positional or keyword, and _e_ and _f_ being keyword-only.
+
+```Python
+def (a, b, /, c, d, \*, e, f):
```
Comment (tancnle) [inline docs/Splashkit/Extensions/Python Compatibality/Research & Findings/Python Changelog 3.8.md]: **question:** Is the escape character needed? It seems to cause a syntax error.

File: docs/Splashkit/Extensions/Python Compatibality/Research & Findings/Python Changelog 3.8.md [hunk 7]
```diff
@@ -0,0 +1,41 @@
...
+the failure in compatibility.
+
+## Python 3.8
+
+**Assignment expression** introduces new syntax, used by :=, to assign values to variables as part
+of a larger expression.
+
+```python
+if (n := len(a)) > 10: 
+print(f"List is too long ({n} elements, expected <= 10)")
```
Comment (tancnle) [inline docs/Splashkit/Extensions/Python Compatibality/Research & Findings/Python Changelog 3.8.md:22]:
```suggestion
  print(f"List is too long ({n} elements, expected <= 10)")
```

**Pull request closed without merging**